src/textOnlyPackages/src/microsoft.netcore.platforms/2.1.0/[Content_Types].xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Think this is also usually generated during pack.

Reply:
Removed.